gradio/blocks.py

```diff
@@ -1626,7 +1626,7 @@ def get_time():
                 every=every,
                 no_target=True,
             )
-            return Dependency(self, dep, dep_index)
```

Comment:
Need to handle the the decorator case for `Blocks.load()` as well, right?

Reply:
the Blocks.load method is such a nasty mess right now, we can add this later once we fix the load method in 4.0.